test/public_ecr_repositories/public_ecr_repositories_stack/public_ecr_repositories_stack.py

```diff
@@ -1,38 +1,37 @@
-from aws_cdk import (
-    Stack,
-    aws_ecr as ecr
-)
+from aws_cdk import Stack
+from aws_cdk import aws_ecr as ecr
 from constructs import Construct
 
-class PublicEcrRepositoriesStack(Stack):
 
+class PublicEcrRepositoriesStack(Stack):
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
 
         languages = [
-            'ruby',
-            'javav2',
-            'javascriptv3',
-            'gov2',
-            'python',
-            'dotnetv3',
-            'kotlin',
-            'rust_dev_preview',
-            'swift',
-            'cpp',
-            'sap-abap'
+            "ruby",
```

Comment:
Do you still need this list, if it's in the yaml config? (I guess the accounts config is technically in a different stack?)

If you do still need it, sort the list. (Unless the order matters, in which case, add a load-bearing comment indicating why this is the correct order.)

Reply:
This file was out of scope for the change, but I'll let this one pass bc it's a fine point. Refactored to use shared config and moved shared config to a top-level directory.